docs/reference/commandline/run.md

```diff
@@ -86,17 +86,17 @@ and linking containers.
 ## Examples
```

Comment:
The example doesn't show the `exit` returned.  Also, shouldn't use first person pronouns:
![image](https://cloud.githubusercontent.com/assets/1347209/8533644/3a9d1da6-23eb-11e5-9f84-2ba41e575ad8.png)

--->

    $ docker run --name test -it debian
    root@d6c0fe130dba:/# exit 13
    exit
    $ echo $?
    13
    $ docker ps -a | grep test
    d6c0fe130dba        debian:7            "/bin/bash"         26 seconds ago      Exited (13) 17 seconds ago                         test


This example runs container named `test` using the `debian:latest` image. 
The `-it` flags Docker to immediately open a `bash` shell in the container. In
the example, the `bash` shell is quit by entering `exit 13` in the shell. This
exit code is passed on to the caller of `docker run`, and is recorded in the
`test` container's metadata.